upup/pkg/fi/cloudup/containerd.go

```diff
@@ -185,7 +185,7 @@ func findAllContainerdDockerMappings() map[string]string {
 	versions := map[string]string{
 		"1.3.7": "19.03.13",
 		"1.3.9": "19.03.14",
-		"1.4.3": "20.10.0",
+		"1.4.3": "20.10.5",
```

Comment:
This should not change unless there is a known packaging bug.
```suggestion
		"1.4.3": "20.10.0",
```